### PR TITLE
fixed speak text button and made it responsive.

### LINKED ISSUE #727: Speak Text button hides text under it in speak activity
Screenshot from 2020-03-21 23-40-21

### PR DESCRIPTION
**Pull request by Swayamsvk**
fixed #727 
speak1

### CODE REVIEW

**Comment by Swayamsvk** [comment]
@llaske please review this PR

File: activities/Speak.activity/css/activity.css [hunk 1]
```diff
@@ -35,25 +35,71 @@
...
+	background-color: white;
+	border: 2px solid white;
+  }
+  
+  .search-field-border-nofocus {
+	background-color: #e5e5e5;
+	border: 2px solid #e5e5e5;
+  }
+
+  /* #userText {
```
Comment (llaske) [inline activities/Speak.activity/css/activity.css]: Do not let commented code

File: activities/Speak.activity/index.html [hunk 2]
```diff
@@ -53,10 +53,13 @@
 		<!-- The select tag acts like a drop down button, so it passes its value to the input box and not to itself -->
 		<select id = 'combo-box' title = "Saved Talk" onchange="document.getElementById('userText').value=this.options[this.selectedIndex].text; document.getElementById('idValue').value=this.options[this.selectedIndex].value;">
 	  </select>
-		<span class = "dropdown" name = "Saved Talk"></span>
-		<input id ="userText" name="userText" type="text" onfocus="this.select()" ></input>
+    <span class = "dropdown" name = "Saved Talk"></span>
```
Comment (llaske) [inline activities/Speak.activity/index.html]: Avoid mixing indentation style in files. Here it will be better to use TAB

**Comment by Swayamsvk** [comment]
@llaske I have fixed the indentation please review this PR and if possible merge this

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**